CONTRIBUTING.md

```diff
@@ -1,8 +1,19 @@
+# Contributing
+
+## Getting Started
+
+Contributions are made to this repo via Issues and Pull Requests (PRs). A few general guidelines that cover both:
+
+- Search for existing Issues and PRs before creating your own.
+- We work hard to makes sure issues are handled in a timely manner but, depending on the impact, it could take a while to investigate the root cause. A friendly ping in the comment thread to the submitter or a contributor can help draw attention if your issue is blocking.
+
+## Issues
+
 All kinds of patches are welcome. Feel free to even suggest some catchy and funny titles for the existing Examples. The goal is to make this collection as interesting to read as possible. Here are a few ways through which you can contribute,
 
-- If you are interested in translating the project to another language (some people have done that in the past), please feel free to open up an issue, and let me know if you need any kind of help.
+- If you are interested in translating the project to another language, please feel free to open up an issue using `translation` template, and let me know if you need any kind of help.
```

Comment:
Sorry I might have forgotted, Is the `translation` template already merged.

Reply:
No, it stored in my local `git stash` for now, I will add it to current PR a bit later, just waited for your comments about the whole PR